spec/policies/patient_policy_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe PatientPolicy do
+  let(:user) { create(:user) }
+  let(:patient) { create(:patient, user: user) }
+  let(:patient_without_user) { create(:patient) }
+
+  describe PatientPolicy::Scope do
```

Comment:
What is missing in this spec?

Reply:
owner context

Reply:
Done! f9c127650d08c779d0ac2c8c204fc2c96036c006